static/src/js/components/CollectionDetails/CollectionDetailsCloudAccessInstance.js

```diff
@@ -0,0 +1,168 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import './CollectionDetailsCloudAccessInstance.scss'
+
+export const CollectionDetailsCloudAccessInstance = ({
+  type,
+  instanceInformation
+}) => {
+  if (type === 'variable') {
+    const {
+      url, format, description, directDistributionInformation
+    } = instanceInformation
+
+    const {
+      region,
+      s3BucketAndObjectPrefixNames = [],
+      s3CredentialsApiEndpoint,
+      s3CredentialsApiDocumentationUrl
+    } = directDistributionInformation
+    return (
+      <>
+        <div className="collection-details-cloud-access-variable__instance-field">
+          <p className="collection-details-cloud-access-variable__instance-field-title">URL:</p>
+          {' '}
```

Comment:
I'm not sure why all these empty spaces are in this file. 

I'm also not sure that the field names, like `URL:`, should be within a `p` tag.